client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -255,19 +255,17 @@ public void start() throws MQClientException {
     }
 
     private void startScheduledTask() {
-        if (null == this.clientConfig.getNamesrvAddr()) {
```

Comment:
@Jaskey agree with you , and my question is if user does not specify an namesrv and its purpose is fetch from wsAddr , but client can not fetch namesrvs dymatically , see 

// If not specified,looking address from name server 
if (null == this.clientConfig.getNamesrvAddr()) { this.mQClientAPIImpl.fetchNameServerAddr(); }
 
private void startScheduledTask() {
      if (null == this.clientConfig.getNamesrvAddr()) {.......}